tests/robustness/linearizability_test.go

```diff
@@ -102,12 +102,14 @@ func TestRobustness(t *testing.T) {
 			e2e.WithClusterSize(1),
 		),
 	})
-	if v.Compare(version.V3_5) >= 0 {
+	// TODO: Deflake waiting for waiting until snapshot for etcd versions that don't support setting snapshot catchup entries.
```

Comment:
How about? 

```
// Disable BlackholeUntilSnapshot failpoint for etcd versions that don't support setting smaller snapshot catchup entries. Default is 5000 which is too long to wait in robustness test. 
```

The reason I propose removing TODO is I don't see how to address the TODO, either wait longer for 5000 entries or externalize snapshot catchup entries as etcd flags seems not actionable right now.

Reply:
Still we should address it, I would even consider backporting the flag (maybe hidden) just to be able to test for regressions.